README.md

```diff
@@ -6,201 +6,143 @@
-```
-Using --conftest-seed=1691289424
-===== test session starts =======
-platform linux -- Python 3.11.2, pytest-7.2.0, pluggy-1.3.0
-Using --randomly-seed=1956010105
-```
-Copy and paste previously output seeds (or type them out) as the arguments to repeat results.
+CURRENT DATA: [946707838.0, 30.0, 12.0, 200.0, 0.0, 0.0, 182.1, 0.0, 1.0]
+INTERPRETED SYSTEM STATUS: ---
 
-### Running pytest directly from command line
+--------------------- STEP 1440 ---------------------
 
-For the equivalent of the driver.py run:
-```
-python -m coverage run --branch --source=onair,plugins -m pytest ./test/
+CURRENT DATA: [946707839.0, 30.0, 12.0, 200.0, 0.0, 0.0, 182.28, 0.0, 1.0]
+INTERPRETED SYSTEM STATUS: ---
 ```
-
-#### Command breakdown:
-
-`python -m` - invokes the python runtime on the library following the -m  
-`coverage run` - runs coverage data collection during testing, wrapping itself on the test runner used  
-`--branch` - includes code branching information in the coverage report  
-`--source=onair,plugins` - tells coverage where the code under test exists for reporting line hits  
-`-m pytest` - tells coverage what test runner (framework) to wrap  
-`./test` - run all tests found in this directory and subdirectories  
-
-#### A few optional settings for the command line
-Options that may be added to the command line test run. Use these at your own discretion.
-
-`--disable-warnings` - removes the warning reports, but displays count (i.e., 124 passed, 1 warning in 0.65s)  
-`-p no:randomly` - ONLY required to stop random order testing IFF pytest-randomly installed  
-`--conftest-seed=###` - set the random values seed for this run  
-`--randomly-seed=###` - set the random order seed for this run  
-`--verbose` or `-v` - set verbosity level, also -vv, -vvv, etc.  
-`-k KEYWORD` - only run tests that match the KEYWORD (see `pytest --help`)  
-
-NOTE: see note about seeds in driver.py section above
-
-### To view testing line coverage after run:
-NOTE: you may or may not need the `python -m` to run coverage report or html
-
-`coverage report` - prints basic results in terminal  
-or  
-`coverage html` - creates htmlcov/index.html, automatic when using driver.py for testing  
-
-then
-`<browser_here> htmlcov/index.html` - browsable coverage (i.e., `firefox htmlcov/index.html`)
-
-## Running with Core Flight System (cFS)
-OnAIR can be setup to subscribe to and recieve messages from cFS. For more information see [doc/cfs-onair-guide.md](doc/cfs-onair-guide.md)
-
-## License and Copyright
-
-Please refer to [NOSA GSC-19165-1 OnAIR.pdf](NOSA%20GSC-19165-1%20OnAIR.pdf) and [COPYRIGHT](COPYRIGHT).
-
-## Contributions
+At this point it should stop. This default used a CSV file as the data source and stops at end of file. The configuration file used is [default_config.ini](https://github.com/nasa/OnAIR/blob/main/onair/config/default_config.ini). When this runs without errors, your basic setup is complete.
+
+### 5. Next steps:
```

Comment:
This section should point to the doc/README.md

Reply:
Agreed.